### DISCUSSION TITLE
Using Textual through Ubuntu/Linux Distro on a Windows Machine/VS Code?

### DISCUSSION THREAD

**Discussion by schlopp96** [Textualize/textual#73] [Q&A] [answered]
As I understand, **_Textual_** is currently only supported through Linux and MacOS... This is unfortunate (for now) as my default coding-environment utilizes Visual Studio (_Code & 2019_) running on Windows 10.

This got me wondering if perhaps I could somehow just install **_Textual_** with perhaps a Linux distro such as Ubuntu, and still be able to use my other Windows-based extensions/frameworks simultaneously. I don't know _too much_ about cross-OS development or if it's even possible at all, so please forgive my ignorance...  I just **really** want to give this framework a go!

### So basically my question is:

Would it be possible to use Textual in Windows if it's installed & run from Ubuntu/Other Linux Distro, while _still_ being able to utilize my normally installed Windows extensions/frameworks/libraries/ etc. in Visual Studio Code?

**Comment by willmcgugan** [comment]
Have a look in to "virtual machines". You can run Linux in a window on your Windows machine.

**Reply by schlopp96** [reply]
So, are you saying running Linux with WSL2 on Windows through Ubuntu doesn't count as a VM then right? That's what I thought I could've done, that way I wouldn't have to have more than one window open, and just switch between Linux-Ubuntu & Windows-Powershell/Bash terminals within VS Code's integrated terminal.

It seems like downloading a VM to run Ubuntu through that is just adding an extra step, but again I know very little about VMs specifically.

**Comment by schlopp96** [comment, marked as answer]
No longer relevant as **Textual _officially_ supports Windows!**

Thanks @willmcgugan for your hard work and for the great software you produce!
